src/services/eosio/transaction.ts

```diff
@@ -19,13 +10,30 @@ type ActionData = {
     }[];
     account?: string;
     name: string;
-    data: any;
+    data: object;
 };
 
 interface Signer {
     sign(digest: Checksum256): Promise<Signature>;
 }
 
+interface AntelopePushTransactionErrorConstructor extends Error {
```

Comment:
delete this. use `AntelopePushTransactionError` instead like you did for `HttpError`

Reply:
So I should replace `AntelopePushTransactionErrorConstructor` with `AntelopePushTransactionError`, right?